src/crypto/X509Cert.h

```diff
@@ -96,6 +96,7 @@ namespace digidoc
           std::vector<std::string> qcStatements() const;
           bool isCA() const;
           bool isValid(time_t *t = nullptr) const;
+          bool verify(bool noqscd);
```

Comment:
Can you add the tm struct parameter
https://github.com/open-eid/libdigidocpp/blob/master/src/crypto/X509CertStore.h#L51C80-L52C87
and then reuse this in following places, to make sure that it is covered with unittests
https://github.com/open-eid/libdigidocpp/blob/master/src/SignatureXAdES_T.cpp#L117
https://github.com/open-eid/libdigidocpp/blob/master/src/SignatureXAdES_B.cpp#L634

Reply:
Thank you